R/froll.R

```diff
@@ -1,11 +1,42 @@
-froll = function(fun, x, n, fill=NA, algo=c("fast", "exact"), align=c("right", "left", "center"), na.rm=FALSE, hasNA=NA, adaptive=FALSE) {
+partial2adaptive = function(x, n, align) {
+  if (align=="center")
+    stopf("'partial' cannot be used together with align='center'")
```

Comment:
I wonder why? It seems reasonable to me, the spirit of `partial=TRUE` being "shrink the window at the boundaries"; the window will no longer be strictly "centered", but that seems fine.

(I don't expect this support to be added now -- checking my understanding, and possibly generating a future issue to implement)

Reply:
limitation inherited from adaptive=TRUE

Reply:
Could possibly be removed by "ex-post touch-up on behalf of the user for optimal performance?"

The thing is that partial was initially out of scope, then it got into scope with "low hanging fruit & no performance critical" redirect to adaptive.

There are not many users of adaptive functions. I have draft blog post on the topic of adaptive, so once all PRs are in, I could elaborate on use cases there and potentially create some more interest in it.